src/JWT/IJwtDecoder.cs

```diff
@@ -34,6 +43,15 @@ public interface IJwtDecoder
         /// <returns>A string containing the JSON payload.</returns>
         string Decode(string token, byte[] key, bool verify);
 
+        /// <summary>
+        /// Given a JWT, decodes it and return the JSON payload.
+        /// </summary>
+        /// <param name="token">The JWT.</param>
+        /// <param name="keys">The keys bytes provided which one of them was used to sign the JWT.</param>
+        /// <param name="verify">Whether to verify the signature (default is true).</param>
+        /// <returns>A string containing the JSON payload.</returns>
+        string Decode(string token, List<byte[]> keys, bool verify);

```

Comment:
`byte[][] keys` or `IReadOnlyCollection<byte[]` if you will (here and and below)